test/org/sagebionetworks/bridge/dynamodb/DynamoSurveyDaoTest.java

```diff
@@ -75,7 +120,9 @@ public void createPreventsEmptyStudyKey() {
 
     @Test(expected = BridgeServiceException.class)
     public void createPreventsRecreatingASurvey() {
-        surveyDao.createSurvey(testSurvey);
+        Survey survey = surveyDao.createSurvey(testSurvey);
+        surveysToDelete.add(new GuidCreatedOnVersionHolderImpl(survey));
```

Comment:
Discussed this with Alx over IM. The problem is that Survey objects are mutable, so if we save a Survey object and it gets changed (which happens in some of the tests), when we try to delete the survey, we can't find it. We use GuildCreatedOnVersionHolderImpl because it's immutable.